modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

```diff
@@ -27,45 +57,56 @@ spec:
       type: RuntimeDefault
   containers:
   - name: qos-demo-ctr
-    image: <image-pull-spec>
+    image: quay.io/openshifttest/hello-openshift:openshift <1>
     resources:
       limits:
-        memory: "200Mi"
-        cpu: "1"
+        memory: "200Mi" <2>
+        cpu: "1" <3>
       requests:
-        memory: "200Mi"
-        cpu: "1"
+        memory: "200Mi" <4>
+        cpu: "1" <5>
     securityContext:
       allowPrivilegeEscalation: false
       capabilities:
         drop: [ALL]
 ----
+<1> This example uses a public `hello-openshift` image. Replace with the image that you want to use for your workload.
+<2> Sets the memory limit to 200MB.
```

Comment:
```suggestion
<2> Sets the memory limit to 200 MB.
```
In the example YAML, the value is 200 Mi, which I think maybe should be MiB? I found this reference in the "Units of measurement" topic in IBM Style:
>MB
When referring to hard disk drive capacity or communications volume, 1,000,000 bytes.

>MiB
When referring to processor storage, real and virtual storage, or channel volume, 1,048,576 bytes.